web/src/app/users/UserContactMethodList.js

```diff
@@ -43,6 +46,7 @@ export default class UserContactMethodList extends React.PureComponent {
   state = {
     edit: null,
     delete: null,
+    isVerifyDialogOpen: false,
```

Comment:
I prefer more verbose flag names when using booleans because when looking at a file for the first time, I don't really know what `edit` or `delete` actually does without spending extra time looking through the file. I would be okay with altering the edit/delete names instead. I've always posed my booleans as `isSomethingTrue` or `hasSomething` so it's clear that what someone is looking at is a boolean.

Reply:
I'd be fine with updating names to indicate what they are. I suggest `editContactMethodID`, `deleteContactMethodID`, and `verifyContactMethodID`.